app/src/androidTest/java/com/github/sdp_begreen/begreen/activities/DatabaseActivityTest.kt

```diff
@@ -99,11 +99,16 @@ class DatabaseActivityTest {
             .check(matches(withText("email@example.com")))
     }
 
+    /*
+
+    TODO : ce test qui ne passe pas sur la CI ne test rien
```

Comment:
Try keeping comments in English

Reply:
It was a comment for me that I supposed to remove and I forgot. Thank you for pointing it out, I will remember to remove it.

Reply:
Actually, this file was useless and we removed it. But thanks, I'll be careful next time.